clients/bcexporter/connectors/ChainUrl.py

```diff
@@ -1,17 +1,16 @@
 import ipaddress
 import re
-
 from config.Config import Config
 
 
-def get_alias() -> str:
-    return Config().alias
-
-
 class ChainUrl:
 
-    def __init__(self, endpoint):
+    def __init__(self, endpoint, alias=None):
         self.endpoint = endpoint
+        self.alias = Config().alias
```

Comment:
This still reads config file everytime a chain url is constructed. Up the call stack tree, pass in the alias when you create the connectors.

Reply:
resolved